lib/node_modules/@stdlib/lapack/base/dlaswp/README.md

```diff
@@ -0,0 +1,248 @@
+
+-   **order**: storage layout.
+-   **N**: number of columns of `A`.
+-   **A**: input `Float64Array`.
+-   **LDA**: leading dimension of `A`.
+-   **k1**: index of first row to interchange.
+-   **k2**: index of last row to interchange.
+-   **IPIV**: vector of pivot indices.
+-   **sipiv**: index increment for `IPIV`.
+
+The increment parameters determine how elements in the IPIV array are accessed at runtime. For example, to iterate over the elements of `IPIV` in reverse order,
+
+```javascript
+var Int32Array = require( '@stdlib/array/int32' );
+var Float64Array = require( '@stdlib/array/float64' );
+
+var IPIV = new Int32Array( [ 2, 0, 1 ] );
+var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+
+dlaswp( 'column-major', 3, A, 2, 0, 2, IPIV, 2 );
+// A => <Float64Array>[ 3.0, 2.0, 5.0, 4.0, 1.0, 6.0 ]
+```
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Int32Array = require( '@stdlib/array/int32' );
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial arrays...
+var IPIV = new Int32Array( [ 2, 0, 1 ] );
+var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+
+// Create offset views...
+var IPIV1 = new Int32Array( IPIV.buffer, IPIV.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+var A1 = new Float64Array( A.buffer, A.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+
+dlaswp( 'column-major', 3, A1, 2, 0, 2, IPIV1, 1 );
+// A1 => <Float64Array>[ 2.0, 3.0, 4.0, 5.0, 6.0 ]
+```
+
+#### dlaswp.ndarray( order, N, A, oa, LDA, k1, k2, IPIV, sipiv )
+
+Performs a series of row interchanges on the matrix `A` using pivot indices stored in `IPIV` and alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+var Int32Array = require( '@stdlib/array/int32' );
+
+var IPIV = new Int32Array( [ 2, 0, 1 ] );
+var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0 ] );
+
+dlaswp.ndarray( 'row-major', 3, A, 3, 2, 0, 2, IPIV, 1 );
+console.log( A );
+// A => <Float64Array>[ 1.0, 2.0, 3.0, 6.0, 7.0, 4.0, 5.0, 8.0, 9.0 ]
+```
+
+The function has the following additional parameters:
+
+-   **oa**: starting index for `A`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices. For example,
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+var Int32Array = require( '@stdlib/array/int32' );
+
+var IPIV = new Int32Array( [ 2, 0, 1 ] );
+var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0 ] );
+
+dlaswp.ndarray( 'column-major', 3, A, 2, 2, 0, 2, IPIV, 1 );
+console.log( A );
+// A => <Float64Array>[ 1.0, 2.0, 4.0, 5.0, 6.0, 7.0, 3.0, 8.0, 9.0 ]
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   `dlaswp()` is a LAPACK routine.
```

Comment:
```suggestion
-   `dlaswp()` is a [LAPACK]() routine.
```

I think we should have something like this, I am not sure though

Reply:
Yes, and similar to BLAS routines in `stdlib`, you should link to the reference LAPACK docs.